src/compiler/crystal/tools/print_hierarchy.cr

```diff
@@ -95,6 +94,41 @@ module Crystal
       false
     end
 
+    def must_print?(type : NonGenericClassType)
+      !(@exp && !@targets.includes?(type))
+    end
+
+    def must_print?(type : GenericClassType)
+      !(@exp && !@targets.includes?(type))
+    end
+
+    def must_print?(type)
+      false
+    end
+
+    def type_size(type)
+      @llvm_typer.size_of(@llvm_typer.llvm_struct_type(type))
+    end
+
+    def ivar_size(ivar)
+      @llvm_typer.size_of(@llvm_typer.llvm_embedded_type(ivar.type))
+    end
+  end
+
+  class TextHierarchyPrinter < HierarchyPrinter
+    private getter io
```

Comment:
Minor/Style: I don't see much benefit of a private getter method vs. directly reading from the ivar. IMO the latter may even be preferable because it clearly communicates this accesses an instance variable.

Reply:
There are 15 other instances of `private getter` in this repo, by the way.